tests/Resources/OCP.resource

```diff
@@ -47,16 +47,7 @@ Verify CR Status
     &{cr_obj_dictionary} =  Set Variable  ${cr_obj}[0]
     Should Match  ${cr_obj_dictionary.metadata.name}  ${cr_name}
     Should Be Equal  ${cr_obj_dictionary.status.conditions[0].type}  Available
-    Should Be Equal  ${cr_obj_dictionary.status.conditions[0].status}  True
-
-Namespace Should Not Exist
```

Comment:
it would need a refactor because the original version wasn't working correctly. I had to use a kw implemented in DSProjects robot files. Not worth now spending time on it